src/api/plan.ts

```diff
@@ -2,3 +2,78 @@ import express from "express";
 
 const router = express.Router();
 export default router;
+
+
+//Returns a list of plans for the current user.
+router.get('/plan', (req, res) => {
+  res.send({
+    plan_id: 1,
+    program_id: 4505,
+    program_type: "MS",
+    concentration_id: 3
+  });
+});
+
+
+// POST /plan/ - Creates a new plan for a specified program
+router.post('/plan', (req, res) => {
+  res.send({
+    "plan_id": 5
+  });
+});
+
+//Gets a plan for the current user by the plan id.
+router.get('/plan/:plan_id',(req,res) =>{
+  res.send({
+    plan_id: 1,
+  });
```

Comment:
The response should be something more like this:
{
  "program_id": number,
  "program_type": string,
  "concentration_id": number,
}